e2e/updatecli.d/deprecated.d/transformers.yaml

```diff
@@ -1,5 +1,9 @@
 name: "Test transformers"
 
+scms:
+  local:
+    disabled: true
```

Comment:
How about `enabled: false` instead? Seems more common to have a "positive" expression as field name

Reply:
Depends on the languages 😅
- Human language: depends on the culture and people profiles and there isn't really a "best approach".
- Technical language (see below) with zero-ed value make it easier to have a "disabled" attribute

The idea is the following:

- Default behavior is "feature" is enabled
- You want to provide a switch to have disable it if required
- The "zero" value for a boolean in Golang (same problem as helm templates or Groovy language) is `false`. 
- If you want an "enable", then you have to write code for all initialization path that would check the inverted value, since it would be "enable:  false` by default. Much more code to write.

Reply:
You're lazy, I understand 😄 😉